apk-generator/v2/app/views/api.py

```diff
@@ -12,6 +11,15 @@
 
 TASK_RESULTS = {}
 
+@api.route('/app/send-notif', methods=['POST', ])
```

Comment:
Lets call this route,  `/_internal/socket/message` so in future we can deny access to all `/_internal/*` routes to the public.